ambry-api/src/main/java/com.github.ambry/clustermap/ClusterMap.java

```diff
@@ -122,6 +122,14 @@
    */
   JSONObject getSnapshot();
 
+  /**
+   * Attempt to get new replica of certain partition that resides on given data node.
```

Comment:
This is actually "new replica" that will be added to specified datanode. Currently, it is used in HelixClusterManager only because target node needs replica detailed info to create it. The replica info is stored in Helix PropertyStore. Before moving replica, we use Helix Bootstrap tool to upload new replica infos onto Helix PropertyStore, thus target node can fetch this info to create new replica. Let me know if you come up with a better name to disambiguate in this context. Thanks